src/serve/functions.ts

```diff
@@ -9,6 +9,7 @@ import { parseRuntimeVersion } from "../emulator/functionsEmulatorUtils";
 import { needProjectId } from "../projectUtils";
 import { getProjectDefaultAccount } from "../auth";
 import { Options } from "../options";
+import * as functionsConfig from "../functions/projectConfig";
```

Comment:
I find this rename confusing. Based on the file name, it should be `projectConfig`

```suggestion
import * as projectConfig from "../functions/projectConfig";
```